src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Tooltip/RazorClassifiedTextRunStyle.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Tooltip
+{
+    /// <summary>
+    /// Equivalent to VS' ClassifiedTextRunStyle. The class has been adapted here so we
+    /// can use it for LSP serialization since we don't have access to the VS version.
+    /// Refer to original class for additional details.
+    /// </summary>
+    [Flags]
+    public enum RazorClassifiedTextRunStyle
+    {
+        Plain = 0b_0000_0000,
+        Bold = 0b_0000_0001,
```

Comment:
Is this an open classification spec? If not we might call this `VSClassifiedTextRunStyle` instead.